CAM/ShakeGameplayCam.md

```diff
@@ -8,22 +8,101 @@ ns: CAM
 void SHAKE_GAMEPLAY_CAM(char* shakeName, float intensity);
 ```
 
-```
-Possible shake types (updated b617d):  
-DEATH_FAIL_IN_EFFECT_SHAKE  
-DRUNK_SHAKE  
-FAMILY5_DRUG_TRIP_SHAKE  
-HAND_SHAKE  
-JOLT_SHAKE  
-LARGE_EXPLOSION_SHAKE  
-MEDIUM_EXPLOSION_SHAKE  
-SMALL_EXPLOSION_SHAKE  
-ROAD_VIBRATION_SHAKE  
-SKY_DIVING_SHAKE  
-VIBRATE_SHAKE  
+Shakes the gameplay cam with the specified shake name and intensity.
+
+```cpp
+enum eCameraShakes
```

Comment:
Honestly, most other places which utilize enums do so with some form of number value attached to it as the parameter value where here it is a string parameter, in this current state using an enum could suggest that there is some tie to a numeric value when in reality there are just string inputs. 

Personally, I would leave this as a list of strings in a code block for styling.

Reply:
sounds good, fixed